compiler/package.json

```diff
@@ -42,7 +42,8 @@
     "postcss": "6.0.17",
     "postcss-discard-duplicates": "2.1.0",
     "postcss-js": "1.0.1",
-    "postcss-nest-atrules": "0.1.3"
+    "postcss-nest-atrules": "0.1.3",
+    "postcss-reduce-idents": "^4.0.1"
```

Comment:
it seems that this plugin is deprecated now

Reply:
No, it has just been moved to the cssnano monorepo.